### DISCUSSION TITLE
Remove left-facing arrows from Argon Pro skin in Taiko

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22243] [Ideas]
As seen here, basically every Taiko skin has either nothing in the middle of the notes or something subtle, like a circle
https://skins.osuck.net/skins/category/taiko/

The left facing arrow is unnecessary visual information that can be distracting, especially considering how it's a relatively complex shape that also contrasts heavily with the rest of the note due to being pure white against pure black
![osu_2023-01-16_21-02-13-1](https://user-images.githubusercontent.com/48618519/212765290-529ac46d-e617-4ced-b740-cce0c4287d9a.jpg)

I instead propose something like this, as a circle is a significantly simpler shape and fits with the existing shape of the note as to not stand out, however you can do whatever you see fit
![image](https://user-images.githubusercontent.com/48618519/212766000-be4a5d93-49eb-4a1d-8970-9e197e049c1a.png)

**Comment by TheFearlessDeath500** [comment]
Agreed. The arrows feel more like an indicator of what hand you need to use for alternating or something (which no one really uses) so setting the skin like this would be optimal.
